lib/aiken/math.ak

```diff
@@ -208,15 +208,14 @@ test log_4_43() {
 /// math.gcd(14, 42) == 14
 /// math.gcd(0, 0) == 0
 /// ```
-pub fn gcd(a: Int, b: Int) -> Int {
-  do_gcd(abs(a), abs(b))
+pub fn gcd(x: Int, y: Int) -> Int {
+  abs(do_gcd(x, y))
```

Comment:
seems like this simplification is safe off the top of my head. GCD uses modulo, modulo ignores sign.